pkg/cmdlets/checker/checker.go

```diff
@@ -42,3 +31,16 @@ func RunCommitsBehindNotifier(p *project.Project,out output.Outputer) {
 		out.Notice(locale.Tr("runtime_update_help", p.Owner(), p.Name()))
 	}
 }
+
+func CommitsBehind(p *project.Project) (int, error) {
+	latestCommitID, err := model.BranchCommitID(p.Owner(), p.Name(), p.BranchName())
```

Comment:
This should probably have a check to see if we're on a headless commit, and if so we always say 0 commits behind.